scripts/release.py

```diff
@@ -415,8 +475,36 @@ def build_parser():
         default='ssh://git@github.com/rackerlabs/docs-rpc.git',
     )
     parser.add_argument(
+        '--branch', help='Branch to update'
+    )
+    parser.add_argument(
         '--tag', type=Tag, required=True, help='Name of new tag.'
     )
+    parser.add_argument(
+        '--future-tag', type=Tag,
+        help='The version number for the next cycle'
+    )
+    parser.add_argument(
+        '--do-not-publish-release', default=False, action='store_true',
+        help='Do not publish a github release for this release'
+    )
+    parser.add_argument(
+        '--do-not-update-milestones', default=False, action='store_true',
+        help='Do not update github milestones'
+    )
+    parser.add_argument(
+        '--do-not-file-docs-issue', default=False, action='store_true',
+        help='Do not create a docs issue for this release'
+    )
+    parser.add_argument(
+        '--do-not-change-files-with-release-version', default=False,
+        action='store_true',
+        help='Do not update files with the new rpc_release: "future tag"'
+    )
+    parser.add_argument(
+        '--rpc-version-check', default="True",
```

Comment:
I think this should be adjusted. There are different ways you can do this but whichever way, it doesn't make sense to accept any value.